data_collection/other_integrations/web_log_analytics.rst

```diff
@@ -1,14 +1,18 @@
 .. highlight:: bash
 .. default-domain:: bash
 .. _data-collection-web-log-analytics:
+.. _GitHub: https://github.com/PiwikPRO/log-analytics/
+
 
 Web Log Analytics
 =================
 
+
+Log analytics is a python script, that allows importing common web server log formats (nginx, apache, iss and more) directly to Piwik PRO. It's Free Software available under GPLv3 license, available on `GitHub`_
```

Comment:
```suggestion
Log analytics is a python script, that allows you to import logs of common web servers (nginx, apache, iss and more) directly to Piwik PRO. It's a free software available under GPLv3 license, available on `GitHub`_
```
Reason for the first suggestion: this compound noun was too complex - should be split.
Reason for the second suggestion: I'm not 100% sure but IMO the article should be there + I don't see the reason for the capital letters.